src/nvme/ioctl.h

```diff
@@ -5122,12 +5122,56 @@ static inline int nvme_lm_migration_send(nvme_link_t l, __u64 offset, __u16 mos,
 /**
  * nvme_lm_migration_recv - Migration Receive command
  * @l:		Link handle
- * @args:	&struct nvme_lm_migration_rev_args argument structure
+ * @offset:	Offset: This field specifies the offset, in bytes, within the data available to be
+ *		returned and specifies the starting point for that data for what is actually
+ *		returned to the host.
+ * @mos:	Management Operation Specific (MOS): This field is specific to the SEL type
+ * @cntlid:	Controller ID: This field specifies the identifier of the controller to which the
+ *		operation is performed.
+ * @csuuidi:	Controller State UUID Index (CSUUIDI): A non-zero value in this field specifies the
+ *		index to a specific entry in the Vendor Specific Controller State UUID Supported.
+ *		list of the Supported Controller State Formats data structure.
+ * @sel:	Select (SEL): This field specifies the type of management operation to perform
+ * @uidx:	UUID Index (UIDX): If this field is set to a non-zero value, then the value of this
+ *		field is the index of a UUID in the UUID List (refer to Figure 320) that is used by
+ *		the command.
+ * @csuidxp:	Controller State UUID Index Parameter (CSUIDXP): This field is vendor specific.
+ * @data:	Pointer to data
+ * @numd:	Number of Dwords (NUMD): This field specifies the number of dwords to return. This
+ *		is a 0's based value.
```

Comment:
again need to check the spec what's this numd thing means in this context. It looks like the `len` argument from the get log page function but who knows :)

Reply:
I am pretty sure this is the `len` argument. I'll ask someone who should know this detail.

Reply:
Yes I will do change to use len argument and set the NUMDL field by using the argument.